notebooks/roc_plot_example.py

```diff
@@ -1,8 +1,26 @@
 # %% [markdown]
-# This notebook / script shows several use cases of ACDC
-# 
-# (The code relies on our modification of the TransformerLens codebase, 
-# mainly giving all HookPoints access to a global cache)
+# Script of ROC Plots!!!
```

Comment:
⚠️ important file for reference: 

Given a triple (ACDC wandb project, SP wandb project, 16 Heads wandb run), this script produces an ROC plot